docs/docs/adrs/adr-013-equivocation-slashing.md

```diff
@@ -0,0 +1,169 @@
+
+###  Slashing on the provider
+We see that in the single-chain slashing case, we use the `infractionHeight` and the voting `power` to be able to slash.
+In order to slash the provider for a consumer equivocation we need to have the provider's `infractionHeight`
+and voting `power`. However, we do **not** have those values. We only have the `infractionHeight` in the consumer chain
+(that we can extract from the equivocated votes), but we do not know to what provider height this `infractionHeight` corresponds.
+Unless we have a way to find the corresponding `infractionHeight` and `power` in the provider chain, we cannot slash the provider in the same way as we would slash in the single-chain case.
+
+Someone might think that the problem of figuring out the corresponding `infractionHeight` and `power` values in the provider chain is easy because we could have the consumer chain send us this information.
+However, we consider that the application on the consumer chain could be _malicious_ and hence we  cannot really trust anything that stems from the _application state_ of the consumer chain.
+
+Note that when a relayer or a user sends evidence through a [MsgSubmitConsumerDoubleVoting](https://github.com/cosmos/interchain-security/pull/1232/files) message, we get what is contained in the [DuplicateVoteEvidence](https://github.com/cometbft/cometbft/blob/v0.37.0/types/evidence.go#L35):
+```protobuf
+type DuplicateVoteEvidence struct {
+	VoteA *Vote `json:"vote_a"`
+	VoteB *Vote `json:"vote_b"`
+
+	// abci specific information
+	TotalVotingPower int64
+	ValidatorPower   int64
+	Timestamp        time.Time
+}
+```
+The "abci specific information" fields cannot be trusted because they are not signed. Therefore,
+we cannot use `ValidatorPower` in any way for slashing in the provider chain. We can get the `infractionHeight` from the votes, but as already mentioned, this `infractionHeight` corresponds to the infraction height on the consumer and **not** on the provider chain.
+Furthermore, note that the [Timestamp](https://github.com/cometbft/cometbft/blob/v0.37.0/types/vote.go#L55) in the votes is just the [BFT time](https://github.com/cometbft/cometbft/blob/v0.37.0/spec/consensus/bft-time.md), and the time could be anything since a misbehaving validator could have included any time in the votes.
+
+Finally, note that in the single-chain case, we trust the underlying consensus engine (CometBFT) and hence when we receive evidence from CometBFT we can act on it. In the case of provider and consumer chains however we cannot trust the evidence as is (e.g., evidence is submitted by a relayer that could set arbitrary values to the "abci specific information" fields).
+
+
+
+#### Clock drift between the provider and the consumer chain
```

Comment:
I would remove this section from here. For now let's have the discussion on the issue or PR.